eng/common/scripts/Package-Properties.ps1

```diff
@@ -172,6 +179,16 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
         $lookup[$lookupKey] = $pkg
 
         foreach ($file in $targetedFiles) {
+            $shouldExclude = $false
```

Comment:
It might not matter but it may be worth filtering the targetdFiles once outside of this loop to avoid the duplicated work we are doing for every discovered package.

Reply:
@weshaggard, this is the correct place for this. If it's being excluded it needs to skip the rest of the processing in the loop for that particular file.

Reply:
I understand we should skip but we could create a filtered list of the `$targetedFiles` but that is only interesting in cases where we have a large amount of packages or targeted files.